### PR TITLE
Update enum names to keep consistency

### LINKED ISSUE #6339: Not a bug, but an inconsistency, why sorting products enum is named ProductOrderField and other enums are XSortField ?
Well, here's an explanation in ts

```
export enum ProductOrderField {
  /** Sort products by name. */
  Name = 'NAME',
  /** Sort products by price. */
  Price = 'PRICE',
  /** Sort products by a minimal price of a product's variant. */
  MinimalPrice = 'MINIMAL_PRICE',
  /** Sort products by update date. */
  Date = 'DATE',
  /** Sort products by type. */
  Type = 'TYPE',
  /** Sort products by publication status. */
  Published = 'PUBLISHED',
  /** Sort products by publication date. */
  PublicationDate = 'PUBLICATION_DATE'
```
and 
```
export enum OrderSortField {
  /** Sort orders by number. */
  Number = 'NUMBER',
  /** Sort orders by creation date. */
  CreationDate = 'CREATION_DATE',
  /** Sort orders by customer. */
  Customer = 'CUSTOMER',
  /** Sort orders by payment. */
  Payment = 'PAYMENT',
  /** Sort orders by fulfillment status. */
  FulfillmentStatus = 'FULFILLMENT_STATUS',
  /** Sort orders by total. */
  Total = 'TOTAL'
}
```

### PR DESCRIPTION
**Pull request by d-wysocki**
I want to merge this change because...I want to update Enum names to keep consistency in our schema:

`Enums`
- ProductFieldEnum -> ProductField
- CountryCodeEnum -> CountryCode
- LanguageCodeEnum -> LanguageCode
- ProductFieldEnum -> ProductField
- DiscountStatusEnum -> DiscountStatus
- MenuItemsSortField -> MenuItemSortField
- PaymentChargeStatusEnum -> PaymentChargeStatus
- ConfigurationTypeFieldEnum - > ConfigurationType
- ProductOrderField - > ProductSortField
- AddressTypeEnum -> AddressType
- AppTypeEnum -> AppType
- JobStatusEnum - > JobStatus
- AttributeInputTypeEnum -> AttributeInputType
- AttributeEntityTypeEnum -> AttributeEntityType
- AttributeTypeEnum -> AttributeType
- ConfigurationTypeFieldEnum -> ConfigurationTypeField
- CustomerEventsEnum -> CustomerEvents
- ExportEventsEnum -> ExportEvents
- FileTypesEnum -> FileTypes
- OrderEventsEnum -> OrderEvents
- OrderEventsEmailsEnum - > OrderEventsEmails
- WebhookSampleEventTypeEnum -> WebhookSampleEventType
- ShippingMethodTypeEnum -> ShippingMethodType
- WeightUnitsEnum -> WeightUnits
- WebhookEventTypeEnum -> WebhookEventType

`SortingInput`
- ProductOrder - > ProductSortingInput

Fixes #6339

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

**Comment by d-wysocki** [comment]
Added `blocked` label because this PR need to be merged with fronted.

**Review by IKarbowiak — APPROVED** [review]

**Review by IKarbowiak — CHANGES_REQUESTED** [review]
IMO we also need to update `saleor/graphql/core/enums.py` file e.g `JobStatusEnum`, and also `to_enum` method should be updated.

File: saleor/graphql/core/enums.py [hunk 1]
```diff
@@ -85,7 +85,7 @@ def to_enum(enum_cls, *, type_name=None, **options) -> graphene.Enum:
 JobStatusEnum = to_enum(JobStatus)
 PermissionEnum = graphene.Enum("PermissionEnum", get_permissions_enum_list())
 WeightUnitsEnum = graphene.Enum(
```
Comment (IKarbowiak) [inline saleor/graphql/core/enums.py:87]: Why not `JobStatus`, and `WeightUnits` as variable names?

Comment (d-wysocki) [reply saleor/graphql/core/enums.py:87]: The goal of this task was to change enum names to keep consistency in our schema. To change variable name into `JobStatus` I also need to change imports in other places, so I would like to keep this variables name as it is.

**Review by IKarbowiak — APPROVED** [review]

**Review by fowczarek — APPROVED** [review]

**Review by maarcingebala — APPROVED** [review]

**Comment by patrys** [comment]
Task linked: [SALEOR-1512 Schema inconsistency regarding sorting enums](https://app.clickup.com/t/2549495/SALEOR-1512)

**Comment by maarcingebala** [comment]
Closing as there is too many conflicts now, we need to re-create the PR if we want to include these changes.

**Pull request closed without merging**